app/src/main/res/layout/fragment_onboarding_page.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".feature.onboarding.OnboardingPageFragment">
+
+    <ImageView
```

Comment:
Leaving the height to wrap_content might lead to some UI inconsistencies.
Instead, look in the design the height (maybe it's half the screen?) and look into `Barrier`, `Guideline` and `Groups` from the ConstraintLayout API and see which one could be used here.